src/EnergyPlus/HeatBalanceKivaManager.cc

```diff
@@ -299,8 +299,8 @@ void KivaInstanceMap::setInitialBoundaryConditions(
                 int schNameIdCool = state.dataZoneCtrls->TempControlledZone(zoneControlNum).SchIndx_DualSetPointWDeadBandCool;
                 Real64 heatSetpoint = ScheduleManager::LookUpScheduleValue(state, schNameIdHeat, hour, timestep);
                 Real64 coolSetpoint = ScheduleManager::LookUpScheduleValue(state, schNameIdCool, hour, timestep);
-                constexpr Real64 heatBalanceTemp = 10.0; // (assumed) degC
-                constexpr Real64 coolBalanceTemp = 15.0; // (assumed) degC
+                constexpr Real64 heatBalanceTemp = 10.0 + Constant::KelvinConv; // (assumed)
+                constexpr Real64 coolBalanceTemp = 15.0 + Constant::KelvinConv; // (assumed)
```

Comment:
Conditional statements below were comparing degC and degK; now everything is in degK.

Reply:
@nealkruis please confirm this change.  They are tiny so this will be a quick review/test/merge, but you would know if there is any reason not to make this change, and if you need to make any Kiva changes to accompany it.